tests/unit/deferred.py

```diff
@@ -193,9 +196,26 @@ def test_apply_networking(switch, nic1, nic2, network, fresh_database):
                                             channel='vlan/native',
                                             type='modify_port')
 
+    action_native3 = model.NetworkingAction(nic=nic3,
+                                            new_network=None,
+                                            channel='',
+                                            type='revert_port')
```

Comment:
Actually, now that I look at this, I'm not sure this code works -- that check is in the except so it may just be dead code. I don't think we actually validate the information in the networking daemon in a way that would catch this.